tests/mol-audio_exceeding_clipend/META-INF/container.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8" ?>

```

Comment:
SORRRRRRY but test names should have only one underscore (it's a separator between the hyphenated indicator of what requirement we're testing and a descriptor of the test, also possibly hyphenated but usually just one word).

Based on the (unsubmitted) anchor being https://www.w3.org/TR/epub-rs-33/#mol-audio-exceeding-clipEnd, which seems to correspond to "If clipEnd exceeds the full duration of the physical media, Reading Systems MUST assume its value to be the full duration of the physical media," I think this should not be hyphenated at all. I think it should be mol-audio-exceeding-clipend. OK?

(I guess I really haven't explained this properly, but the format is mol-some-identifier-for-the-normative-statement_modifier-if-necessary. Sort of like BEM in CSS (https://css-tricks.com/bem-101/), if you think of the three-letter abbreviation (mol-) as the Block, the normative statement name as the Element, and the optional modifier as the Modifier.)